web/src/pages/task/task-page.tsx

```diff
@@ -19,53 +19,110 @@ import { FlowSideBar } from 'components/task/task-sidebar-flow/task-sidebar-flow
 import { DocumentScale } from 'components/documents/document-scale/document-scale';
 import { PickGridType } from './picker-grid-type/picker-grid-type';
 import { GridVariants } from 'shared/constants/task';
-import { useArrayDataSource } from '@epam/uui';
-import { ReactComponent as goNextIcon } from '@epam/assets/icons/common/navigation-chevron-down-18.svg';
-import { ReactComponent as goPrevIcon } from '@epam/assets/icons/common/navigation-chevron-up-18.svg';
 import { Labels } from 'shared/components/labels';
 import { FlexSpacer } from '@epam/uui-components';
+import { History } from 'history';
+import { DocumentToolbar } from 'shared/components/document-toolbar';
+import { TDocumentPDFRef } from 'shared/components/document-pages/components/document-pdf/types';
+
+type TTaskPageContent = {
+    taskId: string;
+    handleRedirectToNextTask: () => void;
+    history: History<Record<string, string | undefined>>;
+    nextTaskId: number | undefined;
+};
 
-const TaskPage: FC = () => {
+const TaskPageContent: FC<TTaskPageContent> = ({
```

Comment:
One suggestion for improvement would be to move this component to a separate file

Reply:
We can do this as an improvement in the future